src/services/app_service.go

```diff
@@ -0,0 +1,52 @@
+package services
+
+import (
+	"golang-bootcamp-2020/domain/model"
+	"golang-bootcamp-2020/repository/db"
+	"golang-bootcamp-2020/repository/rest"
+	_errors "golang-bootcamp-2020/utils/error"
+)
+
+type Service interface {
+	FetchData(maxPages int) ([]model.Character, _errors.RestError)
+	GetCharacterById(id string) (*model.Character, _errors.RestError)
+	GetAllCharacters() ([]model.Character, _errors.RestError)
+	GetCharacterIdByName(name string) (string, _errors.RestError)
+}
+
+type service struct {
+	restRepo rest.RickAndMortyApiRepository
+	dbRepo   db.DataBaseRepository
+}
+
+func NewService(restRepo rest.RickAndMortyApiRepository, dbRepo db.DataBaseRepository) Service {
+	return &service{
+		restRepo: restRepo,
+		dbRepo:   dbRepo,
+	}
+}
+
+//Fetch data from rest repository and then make csv file
```

Comment:
By convention, all the descriptions should start with the name of the struct, interface, function, etc name
Please change all the descriptions